python/tests/rd_tests/test_rd_sum.py

```diff
@@ -177,9 +177,9 @@ def test_to_from_pandas(summary):
         "TEST", baseline, dims=(smspec.nx, smspec.ny, smspec.nz)
     ).pandas_frame()
 
-    # round to hours
-    roundtrip.index = roundtrip.index.round(freq="H")
-    baseline.index = baseline.index.round(freq="H")
+    # round to days
```

Comment:
Could we leave an explanation here for why we round (so much). Related to some int32 handling of dates?

Reply:
Sure, its because there is so much conversion.

Reply:
Updated

Reply:
not pushed yet (?)

Reply:
Forgot to push, is updated now.